### PR TITLE
Setting Up Police News

### PR DESCRIPTION
**Pull request by tmccoy529**
Using the filter list component I have setup police news. Details doesnt work yet as that is another user story and needs to be setup in SE

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,61 @@
...
-import './App.css';
+import "@baltimorecounty/dotgov-components/lib/styles/dotgov.min.css";
+import "./App.css";
+
+import { Config } from "@baltimorecounty/javascript-utilities";
+import { FilterList } from "@baltimorecounty/react-filter-list";
+import PoliceNewsRoomCard from "./components/PoliceNewsRoomCard";
+import React from "react";
+
+const { setConfig, getValue } = Config;
```
Comment (martypowell) [inline src/App.js:9]: not blocking, but something ot think about, i'd like to have a better way of doing this config part, maybe just dump it in it's own class, or build another method in the JavaScript utils project, so we don't have this 20 liner in every app.js

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,61 @@
...
+
+const testApiRoot =
+  "https://structuredcontentdev.bcg.ad.bcgov.us/api/news/police";
+const prodApiRoot =
+  "https://services.baltimorecountymd.gov/hub/api/news/police";
```
Comment (martypowell) [inline src/App.js:14]: not blocking, this is probably going to have to change, but i can update it.

File: src/components/FilterList.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from "prop-types";
```
Comment (martypowell) [inline src/components/FilterList.jsx:1]: @tmccoy529 I dont think this is being used?

Comment (tmccoy529) [reply src/components/FilterList.jsx]: Yeah youre right. That component is dead. Its been removed

**Pull request merged**